bin/scr_app_list.txt

```diff
@@ -1,4 +1,2 @@
 // This file contains a list of apps need to be updated in some SCR
-Lucidum/Lucidum.json:Lucidum/Lucidum.manifest.json
-Gigamon_HAWK/GigamonHawkApplicationMetadataIntelligence.json:Gigamon_HAWK/GigamonHawkApplicationMetadataIntelligence.manifest.json
-Gigamon_ThreatINSIGHT/GigamonHawkThreatINSIGHT.json:Gigamon_ThreatINSIGHT/GigamonHawkThreatINSIGHT.manifest.json
\ No newline at end of file
+Lucidum/Lucidum.json:Lucidum/Lucidum.manifest.json
```

Comment:
why do we need to include this in scr app list ? we won't need to deploy the app for updating docs.

Reply:
Not required.